pkg/strategy/factorzoo/correlation.go

```diff
@@ -0,0 +1,103 @@
+package factorzoo
+
+import (
+	"fmt"
+	"github.com/c9s/bbgo/pkg/indicator"
+	"github.com/c9s/bbgo/pkg/types"
+	"math"
+	"time"
+)
+
+var zeroTime time.Time
+
+//var zeroTime time.Time
+type KLineValueMapper func(k types.KLine) float64
+
+//go:generate callbackgen -type CORRELATION
+type CORRELATION struct {
+	types.IntervalWindow
+	Values  types.Float64Slice
+	EndTime time.Time
+
+	UpdateCallbacks []func(value float64)
+}
+
+func (inc *CORRELATION) Last() float64 {
```

Comment:
Do you want to define this as a types.Series?

Reply:
how to use series?

Reply:
is there any example ?

Reply:
Implement
- Index(i int) float64
- Length() int
- Last() float64

Refer: pkg/indicators/ewma.go

Reply:
thanks